WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductCard.swift

```diff
@@ -1,7 +1,87 @@
 import Yosemite
 import SwiftUI
 
-struct CollapsibleProductRowCard: View {
+/// Displays a single collapsible product row or grouped parent and child product rows
+struct CollapsibleProductCard: View {
+    @ObservedObject var viewModel: ProductRowViewModel
+
+    /// Used for tracking order form events.
+    let flow: WooAnalyticsEvent.Orders.Flow
+
+    /// Handles when a discount is added to the row with the provided `ProductRowViewModel` ID
+    var onAddDiscount: (Int64) -> Void
+
+    /// Tracks if discount editing should be enabled or disabled. False by default
+    var shouldDisableDiscountEditing: Bool = false
+
+    /// Tracks if discount should be allowed or disallowed.
+    var shouldDisallowDiscounts: Bool = false
+
+    @ScaledMetric private var scale: CGFloat = 1
+
+    init(viewModel: ProductRowViewModel,
+         flow: WooAnalyticsEvent.Orders.Flow,
+         shouldDisableDiscountEditing: Bool,
+         shouldDisallowDiscounts: Bool,
+         onAddDiscount: @escaping (Int64) -> Void) {
```

Comment:
super nit: might help if there's a comment on what the `Int64` is - originally I thought it was the discount amount and wondered why it's an int

```suggestion
         onAddDiscount: @escaping (_ orderItemID: Int64) -> Void) {
```